showTotalLightningBalance.sh

```diff
@@ -59,76 +58,46 @@ PROFIT_AND_LOSS=$((LN_EARNED_FEES_IN_SATS - ONCHAIN_TX_FEES))
-LN_REMOTE_BALANCE_BTC=$(printf %.3f\\n "$(($LN_REMOTE_BALANCE))e-8")
-LN_TOTAL_BALANCE_BTC=$(printf %.3f\\n "$(($LN_TOTAL_BALANCE))e-8")
-LN_INVOICES_BTC=$(printf %.3f\\n "$(($LN_INVOICES))e-8")
-LN_COMMIT_FEES_BTC=$(printf %.3f\\n "$(($LN_COMMIT_FEES))e-8")
-LN_PAYMENTS_BTC=$(printf %.3f\\n "$(($LN_PAYMENTS))e-8")
-LN_PAYMENTS_FEES_BTC=$(printf %.3f\\n "$(($LN_PAYMENTS_FEES))e-8")
+TOTAL_BALANCE_BTC=$(printf %.8f\\n "$((TOTAL_BALANCE))e-8")
+LN_LOCAL_BALANCE_BTC=$(printf %.8f\\n "$((LN_LOCAL_BALANCE))e-8")
+LN_REMOTE_BALANCE_BTC=$(printf %.8f\\n "$((LN_REMOTE_BALANCE))e-8")
+LN_TOTAL_BALANCE_BTC=$(printf %.8f\\n "$((LN_TOTAL_BALANCE))e-8")
 
 ##############################################################################
-# PRINT 
+# PRINT
 ##############################################################################
 
-# turn '0 sats' into '0.000 sats' to keep table nicely formatted
-if [ $LN_LOCAL_BALANCE -eq 0 ]; then
-    LN_LOCAL_BALANCE="0.000"
-fi
-if [ $LN_REMOTE_BALANCE -eq 0 ]; then
-    LN_REMOTE_BALANCE="0.000"
-fi
-if [ $LN_COMMIT_FEES -eq 0 ]; then
-    LN_COMMIT_FEES="0.000"
-fi
-if [ $LN_INVOICES -eq 0 ]; then
-    LN_INVOICES="0.000"
-fi
-if [ $LN_PAYMENTS -eq 0 ]; then
-    LN_PAYMENTS="0.000"
-fi
-if [ $LN_PAYMENTS_FEES -eq 0 ]; then
-    LN_PAYMENTS_FEES="0.000"
-fi
-if [ $ONCHAIN_FUNDS_CONFIRMED -eq 0 ]; then
-    ONCHAIN_FUNDS_CONFIRMED="0.000"
-fi
-if [ $ONCHAIN_FUNDS_UNCONFIRMED -eq 0 ]; then
-    ONCHAIN_FUNDS_UNCONFIRMED="0.000"
-fi
-
 echo -e "\n${YELLOW}LN BALANCE${RESET}"
-echo -e "LOCAL\t\tREMOTE\t\tTOTAL"
-echo -e "--------------\t---------------\t---------------"
-echo -e "$LN_LOCAL_BALANCE sats\t$LN_REMOTE_BALANCE sats\t$LN_TOTAL_BALANCE sats"
-echo -e "$LN_LOCAL_BALANCE_BTC BTC\t$LN_REMOTE_BALANCE_BTC BTC\t$LN_TOTAL_BALANCE_BTC BTC"
-echo -e "$LN_LOCAL_BALANCE_PERCENTAGE%\t\t$LN_REMOTE_BALANCE_PERCENTAGE%\t\t$TOTAL_BALANCE_PERCENTAGE%"
+echo -e "LOCAL             REMOTE            TOTAL           "
+echo -e "----------------  ----------------  ----------------"
+echo -e "$(printf %11s "$LN_LOCAL_BALANCE") sats  $(printf %11s "$LN_REMOTE_BALANCE") sats  $(printf %11s $LN_TOTAL_BALANCE) sats"
+echo -e "$(printf %11s "$LN_LOCAL_BALANCE_BTC") BTC   $(printf %11s "$LN_REMOTE_BALANCE_BTC") BTC   $(printf %11s "$LN_TOTAL_BALANCE_BTC") BTC"
+echo -e "$(printf %11s $LN_LOCAL_BALANCE_PERCENTAGE) %     $(printf %11s $LN_REMOTE_BALANCE_PERCENTAGE) %     $(printf %11s $TOTAL_BALANCE_PERCENTAGE) %"
 
 echo -e "\n${YELLOW}ON-CHAIN BALANCE${RESET}"
-echo -e "CONFIRMED\tUNCONFIRMED\tTOTAL"
-echo -e "--------------\t---------------\t---------------"
-echo -e "$ONCHAIN_FUNDS_CONFIRMED sats\t$ONCHAIN_FUNDS_UNCONFIRMED sats\t$ONCHAIN_FUNDS_TOTAL sats"
-echo -e "$ONCHAIN_FUNDS_CONFIRMED_BTC BTC\t$ONCHAIN_FUNDS_UNCONFIRMED_BTC BTC\t$ONCHAIN_FUNDS_TOTAL_BTC BTC"
-echo -e "$ONCHAIN_FUNDS_CONFIRMED_PERCENTAGE%\t\t$ONCHAIN_FUNDS_UNCONFIRMED_PERCENTAGE%\t\t$TOTAL_BALANCE_PERCENTAGE%"
```

Comment:
not sure how tabs works in console and how to make it "responsive" I am open to suggestions